user-guide/data-sources/google-analytics.md

```diff
@@ -0,0 +1,48 @@
+# Google Analytics Instructions
+
+## Setup The Datasource
+### Create a Service Account
+1. Open the [Service accounts page](https://console.developers.google.com/permissions/serviceaccounts). If prompted, select a project.
+2. Click **Create service account**.
+3. In the Create service account window, type a name for the service account, and select Furnish a new private key. When prompted, select **JSON key file type**. Then click **Create**.
```

Comment:
hmm looks like a better version of how to create a service account than what we have in bigquery page. let's standartize on one of them and reuse in both places.